### PR TITLE
chore(helm): updating the litmus-portal manifests

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

### CODE REVIEW

**Comment by ksatchit** [comment]
can we also pull in the changes corresponding to our admin/ns mode

File: charts/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -33,3 +33,20 @@ subjects:
   name: {{ include "litmus.fullname" . }}-admin
   namespace: {{ .Release.Namespace }}
 {{ end }}
+{{ if .Values.litmusPortal.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: litmus-rb
```
Comment (uditgaurav) [inline charts/templates/clusterrolebinding.yaml:41]: Should it be `name: litmus`?
Ref: https://mayadata-team.slack.com/archives/C010QVBA10A/p1597491206079500

Comment (ispeakc0de) [reply charts/templates/clusterrolebinding.yaml:41]: I take refer from our litmus-portal manifest and try to sync with this: https://github.com/litmuschaos/litmus/blob/master/litmus-portal/k8s-manifest.yml#L169

**Comment by ispeakc0de** [comment]
> can we also pull in the changes corresponding to our admin/ns mode

Updated!

File: charts/values.yaml [hunk 2]
```diff
@@ -111,6 +111,10 @@ litmusPortalServer:
         tag: ci
```
Comment (ksatchit) [inline charts/values.yaml:111]: values.yaml needs to change as per latest changes upstream

Comment (ispeakc0de) [reply charts/values.yaml:111]: I took refernce from https://github.com/litmuschaos/litmus/blob/master/litmus-portal/k8s-manifest.yml#L64

**Review by ksatchit — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**